### ISSUE TITLE
Upgrade Crossbeam

### ISSUE THREAD

**Issue by pythoneer** [pimeys/a2#31]
Do we have an easy way to upgrade crossbeam to a version >=0.4.1?

As i see it, the current version that is being fetched (3.2) has the "ability" to double-free. Nothing that i am affected by but just as a suggestion. I just get a waring by "cargo-audit" that this version of crossbeam that is pulled by `a2` is affected by this:

https://github.com/RustSec/advisory-db/blob/master/crates/crossbeam/RUSTSEC-2018-0009.toml

https://github.com/crossbeam-rs/crossbeam-epoch/issues/82

**Comment by pimeys** [comment]
Ha, yeah. It's quite easy it seems, at least this compiles with very small changes.

Our company crashed and I have no apple devices now to test, so feel free to try the master branch with https://github.com/pimeys/a2/commit/51ba18a3900e85bd66ad0e3d78cb4e9ccc5209a3

**Comment by pythoneer** [comment]
Works perfect! Thanks for the fast response. Sad to hear that, i hope you're doing well regardless. Do you plan to still maintain this (and related) crates? I am quite happy with the current state of those.

**Comment by pimeys** [comment]
Yes I'll maintain them as a hobby, still planning to go to the async/await syntax with all crates.

Working soon for prisma.io as a Rust engineer, so it worked out quite well.
